script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -68,27 +68,27 @@ Although, we've already met today, so 'good to see you' isn't the right phrase."
 [sha:2d7b1b7315cdafa2d1ef8c5e60cda893dbb11cb2]{
 -- Page 59, Offset 634.
 -- 「あ……うん。そういえば、そうでした」
-"Ah... yeah. Now that you say that, you're right."
+"Oh... yeah. Now that you mention it, true."
```

Comment:
that's true.

Reply:
Suuuure, but that's not very colloquial of you.

Reply:
This one didn't entirely sit right with me either. `True` as a fragment at the end of a sentence with this tone just feels off when I read it, I'd probably render it closer to 
> Oh... yeah. I guess you're right.

Reply:
how about just 'true that'?

Reply:
Much more of a slang / we are in strong agreement vibe than I'd use here.